web/src/features/exchanges/ExchangeArrow.tsx

```diff
@@ -66,22 +68,29 @@ function ExchangeArrow({ data, viewportWidth, viewportHeight, map }: ExchangeArr
   }
 
   return (
-    <div
-      id={key}
-      style={{
-        transform: `translateX(${transform.x}px) translateY(${transform.y}px) rotate(${transform.r}deg) scale(${transform.k})`,
-        cursor: 'pointer',
-        overflow: 'hidden',
-        position: 'absolute',
-        pointerEvents: 'all',
-        imageRendering: 'crisp-edges',
-        left: '-25px',
-        top: '-41px',
-      }}
+    <TooltipWrapper
+      tooltipClassName="relative flex max-h-[256px] max-w-[512px] top-[-76px] rounded border bg-white p-1 px-1  text-sm drop-shadow-sm dark:border-0 dark:bg-gray-900"
+      tooltipContent={<ExchangeTooltip exchangeData={data} />}
+      side="right"
+      sideOffset={10}
     >
-      <source srcSet={`${imageSource}.webp`} type="image/webp" />
-      <img src={`${imageSource}.gif`} alt="" />
-    </div>
+      <div
+        id={key}
+        style={{
+          transform: `translateX(${transform.x}px) translateY(${transform.y}px) rotate(${transform.r}deg) scale(${transform.k})`,
+          cursor: 'pointer',
+          overflow: 'hidden',
+          position: 'absolute',
+          pointerEvents: 'all',
+          imageRendering: 'crisp-edges',
+          left: '-25px',
+          top: '-41px',
+        }}
+      >
+        <source srcSet={`${imageSource}.webp`} type="image/webp" />
+        <img src={`${imageSource}.gif`} alt="" />
```

Comment:
Fixes https://linear.app/electricitymaps/issue/ELE-1481/make-arrow-non-draggable while at it 🥳 
```suggestion
        <img src={`${imageSource}.gif`} alt="" draggable={false} />
```